pricing.js

```diff
@@ -1,36 +1,41 @@
+function volLife(product, price, dollarsOff) {
+  if (product.employerContribution.mode === 'dollar') {
+    price = price - product.employerContribution.contribution
+  } else {
+    dollarsOff = price * (product.employerContribution.contribution / 100)
+    price = price - dollarsOff
+  }
+  return price
+}
+function formatPrice(price) {
+  price = parseInt(price * 100) / 100
+  return price
+}
+function ltdCoverage(price, employee, product) {
```

Comment:
Function names should always tell us what it is doing. `calculateLTDPrice` would be a much better name.